draft-irtf-cfrg-frost.md

```diff
@@ -524,25 +505,25 @@ signature; see {{sec-considerations}}.
 Round one involves each signer generating a pair of nonces and their corresponding public
 commitments. A nonce is a pair of Scalar values, and a commitment is a pair of Element values.
 
-Each signer in round one generates a nonce `nonce = (d, e)` and commitment
-`comm = (D, E)` for each signer.
+Each signer in round one generates a nonce `nonce = (hiding_nonce, blinding_nonce)` and commitment
+`comm = (hiding_nonce_commitment, blinding_nonce_commitment)` for each signer.
```

Comment:
Do you need the "for each signer" specification again at the end of this sentence?